akka-stream/src/main/boilerplate/akka/stream/FanOutShape.scala.template

```diff
@@ -20,12 +21,12 @@ object FanOutShape {
   }
 }
 
-abstract class FanOutShape[I] private (_in: Inlet[I], _registered: Iterator[Outlet[_]], _name: String) extends Shape {
+abstract class FanOutShape[-I] private (_in: Inlet[I @uncheckedVariance], _registered: Iterator[Outlet[_]], _name: String) extends Shape {
```

Comment:
Yes, if they want to define custom shapes from Scala with variance; this should be rarely necessary in general (i.e. with generics), though.